test/test_profiler.py

```diff
@@ -99,6 +101,62 @@ def forward(self, x):
 
         torch._C._set_graph_executor_optimize(prev_opt)
 
+class TestProfilePython(TestCase):
```

Comment:
need to create new class or just add to `TestProfiler`?

Reply:
fixed.